examples/neural_networks/plot_random_nn_overfitting.py

```diff
@@ -0,0 +1,71 @@
+"""
+===========================================================================
+Impact of increasing the number of hidden neurons in random neural networks
```

Comment:
I would remove "increasing". The title is already pretty long. Do you think it would make sense to merge this with the other example?